cloud-common/src/main/java/com/sequenceiq/cloudbreak/domain/CommonStack.java

```diff
@@ -0,0 +1,42 @@
+package com.sequenceiq.cloudbreak.domain;
+
+import java.util.Map;
+
+public interface CommonStack {
```

Comment:
I agree that we move out the enums to common project, but we shall avoid such artificial interface like this CommonStack